server/test/handlers/inputSchedule.test.js

```diff
@@ -0,0 +1,72 @@
+const appointmentHandler = require('../../models/handlers/workerTimeslot');
```

Comment:
The tests are not being run properly as the actual handler file is named `input_schedule` (it might be better to rename the handler file to `schedule` to increase flexibility as it may contain other methods, such as fetching schedules in the future). Remember to keep file names consistent and I would rename the `appointmentHandler` to the appropriate name used below, perhaps `scheduleHandler` to improve clarity of the specific handler.

Reply:
Thanks @catai9 for your comment! I've updated and changed the handler name. Please let me know if this is what you're looking for!